src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/Address.kt

```diff
@@ -27,16 +27,25 @@ import java.time.LocalDate
 @DiscriminatorColumn(name = "OWNER_CLASS")
 @Inheritance
 abstract class Address(
+  @Column(name = "PREMISE")
```

Comment:
Adding the `@Colomn` annotations for consistency - though not needed  - for some reason not having and this being an abstract class is just a little bit confusing to my brain